yinstruments/PDU_main.py

```diff
@@ -0,0 +1,236 @@
+import telnetlib
```

Comment:
Rename this module `pdu.py` instead of `PDU_main.py`.  Also remove the `PDU_CONTROL` folder, and just have `pdu.py` in the top-level package directory.